CONTRIBUTING.md

```diff
@@ -0,0 +1,146 @@
+# Contributing to this project
+
+Please take a moment to review this document in order to make the contribution
+process easy and effective for everyone involved.
+
+Following these guidelines helps to communicate that you respect the time of
+the developers managing and developing this open source project. In return,
+they should reciprocate that respect in addressing your issue or assessing
+patches and features.
+
+
+## Using the issue tracker
+
+The issue tracker is the preferred channel for [bug reports](#bugs),
+[features requests](#features) and [submitting pull
+requests](#pull-requests), but please respect the following restrictions:
+
+* Please **do not** use the issue tracker for personal support requests (use
```

Comment:
I'm unsure how I feel about this.
Cause I'd have to watch stack overflow. Not a problem, just something new.

thoughts?
I'm not sure how new libs deal with this. hmm.

Reply:
I am going to omit this until something formulates.